test/e2e/specs/site-editor/hybrid-theme.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Hybrid theme', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'gutenberg-test-themes/emptyhybrid' );
```

Comment:
Just the theme name without the directory should work here. At least it used to work - https://github.com/WordPress/gutenberg/pull/48343/files#diff-3b4760b18eb814c5a90d5b89fd9394d17350275708fb6196d82e04a74da94f0e.

Reply:
Fixed by [e7025d2](https://github.com/WordPress/gutenberg/pull/52656/commits/e7025d230b62245917e31f1d059aaad8c2d51610)